Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -3670,6 +3670,8 @@ static String commandNameForSelector(SEL selector)
 
 void WebViewImpl::setIgnoresNonWheelEvents(bool ignoresNonWheelEvents)
 {
+    RELEASE_LOG(MouseHandling, "WWebViewImpl::setIgnoresNonWheelEvents(%llu):%d", m_page->identifier().toUInt64(), ignoresNonWheelEvents);
```

Comment:
We need to follow the usual `[pageProxyID=%lld]` format to facilitate grep'ing. We spent a lot of time trying to make our logging consistent, let's be careful to not diverge with new logging.